packages/desktop/src/renderer/index.tsx

```diff
@@ -30,7 +27,20 @@ const persistentConfigService = new PersistentConfigServiceElectron();
 const applicationInfoService = new ApplicationInfoServiceElectron();
 const databaseService = new DatabaseServiceElectron();
 const executionEnvService = new ExecutionEnvServiceElectron(notificationService);
-// application analytics/metrics
+
+// Define the KeyDownHandler component inline
+const KeyDownHandler: React.FC<{ clearStore: () => void }> = ({ clearStore }) => {
+    useKeyDown(["Control", "-"], clearStore);
```

Comment:
Since we're able to customize the app menu options in desktop/src/main/menu, could we expose this command there? Maybe in "View" if possible since that's where Reload, Force reload, and Toggle dev tools are.

Also just to verify, control is the same in both windows and mac right?

Reply:
control is a common key. There could be some weirdness between OS for sure. I tested it with my macbook and with a external keyboard that built for windows and it seemed to be fine.

Reply:
Open to adding it to View!